File: src/main/java/no/nav/melosys/eessi/models/sed/nav/EndringA008.java
package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;
import lombok.Data;

@SuppressWarnings("unused")
@JsonInclude(Include.NON_NULL)
@Data
public class EndringA008 {

    private String periode;

    private Adresse arbeidssted;

    private Adresse adresse;

    private EndringA008Bruker bruker;

    private String trerikraftfra;

    private String startdato;

    private String sluttdato;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/EndringA008.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class EndringA008(
    // Any? fordi CDM 4.3 sender periode som String mens CDM 4.4 sender Periode (objekt med startdato/sluttdato).
    var periode: Any? = null,
    var arbeidssted: Adresse? = null,
    var adresse: Adresse? = null,
    var bruker: EndringA008Bruker? = null,
    var trerikraftfra: String? = null,
    var startdato: String? = null,
    var sluttdato: String? = null
)
